app/views/root/400.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= render partial: "error_page", locals: {
     heading: "Sorry, there is a problem",
     intro: '<p class="govuk-body">Go back and change the information you entered.</p>',
-    status_code: 400
+    status_code: 400,
```

Comment:
I was wondering why there were so many changed files in this PR 😂 Nice one 👍

Reply:
I do love me a dangling comma. Better diffs ftw!